### ISSUE TITLE
Now that NGINX 1.25 has been made stable, when will WordOPs upgrade to 1.25?

### ISSUE THREAD

**Issue by jazir555** [WordOps/WordOps#632]
Nginx 1.25 has gone stable, will WordOPs stack be updated soon?

**Comment by VirtuBox** [comment]
Hello,
Nginx stable release are always even number. Nginx 1.26 is not available yet.
Or are your talking about https://freenginx.org/ ?

**Comment by jazir555** [comment]
> Nginx 1.26

I guess I misread their page and interpreted mainline as stable.  1.24 was released in October 2022, is it possible you could make a second branch with a separate wo command to update to 1.25? I really would like to use HTTP/3, I'm sure it would be a huge performance boost to my sites. I have no idea when 1.26 will release and I'd really like to start using the newer version now.

**Comment by VirtuBox** [comment]
Hello @jazir555,
The next Nginx stable release should be available in the next few weeks. WordOps will support it as soon as it's available but I cannot confirm if HTTP/3 will be available yet. Because currently I'm building Nginx package with OpenSSL library, and it still not support QUIC and I'm not sure to be able to build Nginx with LibreSSL.

**Comment by jazir555** [comment]
@VirtuBox 

According to ChatGPT, building nginx with LibreSSL seems pretty straightforward with relatively few steps. Hope that helps.

**Comment by VirtuBox** [comment]
@jazir555,
yes, I'm also maintaining a script to compile the latest Nginx release with additional modules but I haven't had time to look into the HTTP/3 subject.

**Comment by jazir555** [comment]
Gotcha. Well hopefully you can make the nginx http/3 module work, I hope it will bring a big improvement in performance.

**Comment by VirtuBox** [comment]
@jazir555 HTTP/3 is now available with my script to compile Nginx : https://github.com/VirtuBox/nginx-ee/releases/tag/v3.8.0

**Comment by jazir555** [comment]
How can I upgrade to that new version from the default wordops installation?

**Comment by VirtuBox** [comment]
Nginx stable release v1.26.0 is now available.
You just have to run `bash <(wget -O - vtb.cx/nginx-ee || curl -sL vtb.cx/nginx-ee) --stable --libressl`
But you will have to adjust your SSL conf according to nginx doc : https://nginx.org/en/docs/quic.html
I'm working on update Nginx package for WordOps so it should be available soon.

**Comment by samkatakouzinos** [comment]
> I'm working on update Nginx package for WordOps so it should be available soon.

Would this update also convert existing WordOps websites to QUIC as well, please?!

**Comment by VirtuBox** [comment]
> > I'm working on update Nginx package for WordOps so it should be available soon.
> 
> Would this update also convert existing WordOps websites to QUIC as well, please?!

Yes, I'm working on it to convert existing site to HTTP/3 depending on the Nginx release installed on the server.

**Comment by samkatakouzinos** [comment]
> Yes, I'm working on it to convert existing site to HTTP/3 depending on the Nginx release installed on the server. 

Good! 

I successfully updated an installation with latest nginx-ee with http3 QUIC support.
I unsuccessfully updated the conf files to make http3 QUIC happen. 

So I'm looking forward to this conversion support.

**Comment by jazir555** [comment]
@VirtuBox 

Will the update that auto-enables http/3 be coming soon?

**Comment by VirtuBox** [comment]
HTTP/3 QUIC is now available.

Please update WordOps with the command `wo update`

**Comment by jazir555** [comment]
@VirtuBox 

I updated and ran the migrate command to upgrade to the HTTP/3 version and ran it through a http/3 checker and the checker shows http/3 is active. Most assets are being served via HTTP/3 now. 

However, my jquery is still being served over HTTP/2:

![aa](https://github.com/WordOps/WordOps/assets/31750063/97c85930-3ac5-4c79-aeac-82d3a1b23404)

Any idea why? Can that be fixed so that jquery is served over HTTP/3? Jquery is one of the files that needs it most.
